packages/client/hmi-client/src/types/Types.ts

```diff
@@ -688,7 +688,6 @@ export interface CiemssStatusUpdate {
     loss: number;
     progress: number;
     jobId: string;
-    dataToPersist: any;
```

Comment:
If these are super large do we want them passed up to the front end as well or should they be hit with a `@TSIgnore`? (I'm not sure the context here)

Reply:
Good catch. Fixing

Reply:
Wait nvm the ignore already covered it.

Reply:
We still probably want it on here then that way the field doesn't show up on the front end, even if its empty